tools/hive-bigquery/custom_exceptions.py

```diff
@@ -0,0 +1,27 @@
+"""Defines custom exceptions"""
+
+
+class ConnectionError(Exception):
+    """Raised when there is a connection failure to any of the Hive, BigQuery,
+    GCS, and MySQL components"""
+    pass
+
+
+class MySQLExecutionError(Exception):
+    """Raised when there is a failure in executing query on MySQL database"""
+    pass
+
+
+class HiveExecutionError(Exception):
```

Comment:
Use errors provided by [`pyhive`](https://github.com/dropbox/PyHive/blob/master/pyhive/exc.py)
Also, your doc string is wrong (copied from above I think).

Reply:
This custom exception will handle multiple lower level 'pyhive' errors by grouping them.